app/Http/Controllers/Api/Manager/AttendanceController.php

```diff
@@ -103,22 +104,13 @@ public function store(StoreRequest $request): JsonResponse
      */
     public function show(ShowRequest $request): AttendanceShowResource
     {
+        $instructor = Auth::guard('instructor')->user();
         $courseId = $request->course_id;
 
-        // 現在ログインしているinstructorのidを取得
-        $instructorId = Auth::guard('instructor')->user()->id;
-        // ログインしている講師とその管理している講師を取得
-        $manager = Instructor::with('managings')->find($instructorId);
-        // 管理している講師のIDを配列として取得し、自分自身のIDも追加
-        $instructorIds = $manager->managings->pluck('id')->toArray();
-        $instructorIds[] = $instructorId;
-
         $course = Course::findOrFail($courseId);
-        if (! in_array($course->instructor_id, $instructorIds, true)) {
-            // 自分と配下の講師の講座でない場合はエラーを返す
-            throw new AuthorizationException(
-                'Forbidden, not allowed to access this course.'
-            );
+
+        if (! (new AttendancePolicy)->view($instructor, $course)) {
+            throw new AuthorizationException('Forbidden, not allowed to access this course.');
```

Comment:
次のようにクラスとポリシーのマッピングを明示的に指定できるようになりますので、修正をお願いします！
```suggestion
$this->authorize('view', [Attendance::class, $course]);
```